front/components/app/NewBlock.tsx

```diff
@@ -34,6 +34,13 @@ export default function NewBlock({
       description:
         "Query a Large Language Model using a message-based interface.",
     },
+    {
+      type: "llm",
+      typeNames: ["llm"],
+      name: "Interact with a Large Language Model (LLM)",
+      description:
+        "Query a Large Language Model using a message-based interface.",
+    },
```

Comment:
don't we want to keep llm block hidden ?

Reply:
ah! good catch

Reply:
Updated :+1: